pkg/ig-manager/ig-manager_test.go

```diff
@@ -127,6 +127,17 @@ func checkFdList(t *testing.T, initialFdList string, attempts int, sleep time.Du
 func TestClose(t *testing.T) {
 	utilstest.RequireRoot(t)
 
+	opts := []testutils.Option{
+		testutils.WithName("test-ig-close"),
+		testutils.WithoutRemoval(),
+		testutils.WithoutWait(),
+		testutils.WithoutLogs(),
+	}
+	testutils.RunDockerContainer(context.Background(), t, "sleep inf", opts...)
+	t.Cleanup(func() {
+		testutils.RemoveDockerContainer(context.Background(), t, "test-ig-close")
+	})
```

Comment:
With https://github.com/inspektor-gadget/inspektor-gadget/pull/1850, this will become:

```suggestion
	c := testutils.NewDockerContainer("test-ig-close", "sleep inf", testutils.WithoutLogs())
	c.Start(t)
	t.Cleanup(func() {
		c.Stop(t)
	})
```

Reply:
This looks cleaner :)